gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/bottomsheet/GravatarQuickEditorBottomSheet.kt

```diff
@@ -44,7 +46,33 @@ public fun GravatarQuickEditorBottomSheet(
     authenticationMethod: AuthenticationMethod,
     onAvatarSelected: (AvatarUpdateResult) -> Unit,
     onDismiss: (dismissReason: GravatarQuickEditorDismissReason) -> Unit = {},
-    modalBottomSheetState: SheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true),
+) {
+    val windowHeightSizeClass = currentWindowAdaptiveInfo().windowSizeClass.windowHeightSizeClass
+    GravatarQuickEditorBottomSheet(
+        gravatarQuickEditorParams = gravatarQuickEditorParams,
+        authenticationMethod = authenticationMethod,
+        onAvatarSelected = onAvatarSelected,
+        onDismiss = onDismiss,
+        modalBottomSheetState = if (windowHeightSizeClass == WindowHeightSizeClass.COMPACT) {
+            rememberModalBottomSheetState(
+                skipPartiallyExpanded = true,
+            )
+        } else {
+            rememberModalBottomSheetState(
+                skipPartiallyExpanded = gravatarQuickEditorParams.contentLayout == ContentLayout.Horizontal,
+            )
+        },
```

Comment:
@maxme 
This is what I found to be working. 

Another temporary solution would be to skip the `windowHeightSizeClass == WindowHeightSizeClass.COMPACT`. It's not a dealbreaker. 

As mentioned, it feels like this is a bug. We have no control over the `initial state` so it seems quite wild to crash like that. I've reported it [here](https://issuetracker.google.com/issues/366067966).

Reply:
Thanks for reporting it, I subscribed to the ticket.